CHANGELOG.md

```diff
@@ -8,6 +8,20 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Added
 - Add parsing meshes with vertices for MJCF format ([#2537](https://github.com/stack-of-tasks/pinocchio/pull/2537))
+- Add mimic joint support to the following algorithms: ([#2441](https://github.com/stack-of-tasks/pinocchio/pull/2441))
+  - Forward kinematics
+  - Jacobians and frames
+  - Centroid algorithms (ccrba)
+  - RNEA
+  - CRBA
+  - Reachable workspace
+- Add `mimic` argument in `pinocchio::urdf::buildModel` to parse URDF mimic field (default to false) ([#2441](https://github.com/stack-of-tasks/pinocchio/pull/2441))
+- Add `pinocchio::transformJointIntoMimic` to turn a joint into a mimic joint ([#2441](https://github.com/stack-of-tasks/pinocchio/pull/2441))
+- Add `JointModelBase` methods: ([#2441](https://github.com/stack-of-tasks/pinocchio/pull/2441))
+  - `nvExtended` to get the DoF of the extended model
+  - `idx_vExtended` to get the joint index in the extended vector of velocity
+  - `jointExtendedModel{Cols,Rows,Block}` to get joint columns/rows/block in extended model matrix
+  - `JointMapped{Config,Velocity}Selector` to get mimicked joint configuration/velocity from the mimicking joint
```

Comment:
add line for:
- buildMimicModel
- Model new members
- Data new members
- Change in GeometryModel Scalar value : https://github.com/stack-of-tasks/pinocchio/issues/2445